jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter.bats

```diff
@@ -51,3 +57,61 @@ teardown() {
     unset JAVA_INITIAL_MEM_RATIO
     unset JAVA_MAX_MEM_RATIO
 }
+
+@test "verify the env vars when LOG_LEVEL is set" {
+    export LOG_LEVEL="WARN"
+    PACKAGES_LOG_LEVEL_CONF=org.xyz=INFO,com.acme=DEBUG
+    export PACKAGES_LOG_LEVEL_CONF
+    configure_logger_config_file
+    [[ "${LOG_LEVEL}" == WARN ]]
+    [[ "${PACKAGES_LOG_LEVEL_CONF}" == "org.xyz=INFO,com.acme=DEBUG" ]]
```

Comment:
you just assigned to this env the same value that you're cheking. for what this env is used for?

Reply:
removed was useless